# Motor feedback data as reported

std_msgs/Header header

float64 current         # Measured motor current in A
float64 position        # Measured motor shaft position in rad
